src/components/CountdownTimer.jsx

```diff
@@ -0,0 +1,47 @@
+import React, { useEffect, useState } from "react";
+
+const CountdownTimer = () => {
+  const [, setUpdate] = useState(0);
```

Comment:
:question: Why are we setting this variable if we aren't using it?

Reply:
Discussed on Friday.